user_profile/urls.py

```diff
@@ -1,9 +1,7 @@
-from rest_framework.routers import DefaultRouter
```

Comment:
Refactored `urls` to use view from router with viewset

Reply:
APIView in generics can't use router. Only viewsets can use. Read the docs.

Reply:
yep. so i said "To" generic view, 
and "From" router w viewset